spec/greenhouse_api/client_spec.rb

```diff
@@ -120,4 +120,123 @@
       end
     end
   end
+
+  describe '#create_user' do
+    subject(:create_user) { client.create_user(on_behalf_of, body.to_json) }
+    let(:on_behalf_of) { '12345' }
+
+    context 'when successful' do
+      let(:body) { { first_name: 'Kirby', last_name: 'Birby', email: 'kirby@email.com' } }
+
+      it 'creates user' do
+        VCR.use_cassette('create_user') do
+          response = subject
+          expect(response.status).to eq(201)
+          expect(response.body.dig('id')).to eq(4_314_103_003)
+          expect(response.body.dig('name')).to eq('Kirby Birby')
+          expect(response.body.dig('primary_email_address')).to eq('kirby@email.com')
+          expect(response.body.dig('disabled')).to eq(false)
+          expect(response.body.dig('site_admin')).to eq(false)
+        end
+      end
+    end
+
+    context 'when there is an error' do
```

Comment:
do you know what happens when the on behalf of user is inactive or has invalid permissions?

Reply:
403 (Forbidden – You do not have access to that record) - I can add tests for this case